build.sh

```diff
@@ -34,8 +34,9 @@ rm -rf $P/scratch;
 mkdir -p $P/scratch;
 cd $P/scratch;
 rm -rf $P/install;
+export CFLAGS="-Wall ${werror}"
```

Comment:
Shouldn't we copy over the `-O0` too?

Reply:
Not needed, it come with --enable-debug in configure.